### DISCUSSION TITLE
Can't spectate Multiplayer and Singleplayer games

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16892] [Q&A] [answered]
I joined this room and started spectating, but it's stuck on loading the clients forever even after the match ends.
The room in question could have already been created before the update, but i tried with a new one too with someone on the latest update, and the problem persisted.

https://user-images.githubusercontent.com/53081750/154287965-ebb03e0e-7f0a-4983-8698-dfcb6c8790a0.mp4

This is how single player spectating looks like instead, i've tried to spectate tens of players to make sure that maybe one or two weren't on the pause screen, but it seems that nobody can be spectated.
![image](https://user-images.githubusercontent.com/53081750/154288436-9fb376fd-f894-4ca9-8c03-a3d99ce92718.png)

[performance.log](https://github.com/ppy/osu/files/8080790/performance.log)
[database.log](https://github.com/ppy/osu/files/8080791/database.log)
[network.log](https://github.com/ppy/osu/files/8080792/network.log)

**Comment by peppy** [comment, marked as answer]
The user you're spectating has likely not updated to the new release. Things won't work in this scenario.
